wazo_acceptance/steps/user.py

```diff
@@ -54,18 +54,22 @@ def given_there_are_telephony_users_with_infos(context):
 
         confd_user = context.helpers.confd_user.create(body)
 
+        username = body.get('username') or context.helpers.utils.random_string(10)
+        password = body.get('password') or context.helpers.utils.random_string(10, sample=string.printable)
+        firstname = body['firstname']
+        lastname = body.get('lastname')
```

Comment:
Default value should be an empty string, this breaks other tests.

Reply:
fixed